cmd/runtimetest/main.go

```diff
@@ -588,6 +588,8 @@ func validate(context *cli.Context) error {
 		return err
 	}
 
+	platform := context.String("platform")
```

Comment:
But platform is not a process arg. It should be a generator arg as it's a validate arg.